stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -152,14 +154,14 @@ public class MessageListView : ConstraintLayout, IMessageListView {
         MessageLongClickListenerView { _, view ->
             optionsView = view
 
-            binding.blurLayer.isVisible = true
+            binding.shadowLayer.isVisible = true
 
             if (view.parent != null && view.parent is ViewGroup) {
                 (view.parent as ViewGroup).removeView(view)
             }
 
-            binding.messageOptionsContainer.addView(optionsView)
-            binding.messageOptionsScroll.isVisible = true
+            binding.optionsMessageContainer.addView(optionsView)
+            binding.optionRootContainer.isVisible = true
```

Comment:
The animations produced by `animateLayoutChanges` look really solid 👍

Reply:
Thanks! 😃